core/client/fs/src/main/java/alluxio/client/block/stream/NettyPacketWriter.java

```diff
@@ -256,6 +268,14 @@ public void close() throws IOException {
       if (mChannel.isOpen()) {
         mChannel.pipeline().removeLast();
       }
+      if (closeFuture != null) {
```

Comment:
I meant that we might call `removeLast` on a closed channel, since we are not guaranteed atomicity between the `isOpen` and `removeLast` call (or are you saying we are?)